src/main/java/org/opentripplanner/graph_builder/module/configure/GraphBuilderModules.java

```diff
@@ -54,17 +56,18 @@ static OsmModule provideOpenStreetMapModule(
     Graph graph,
     DataImportIssueStore issueStore
   ) {
-    List<OsmProvider> providers = new ArrayList<>();
-    for (ConfiguredDataSource<OsmExtractParameters> osmConfiguredDataSource : dataSources.getOsmConfiguredDatasource()) {
-      providers.add(
+    var providers = dataSources
+      .getOsmConfiguredDatasource()
+      .stream()
+      .map(source ->
```

Comment:
Please revert this. This is your personal preference, and we do not want "toggle" edits.

Reply:
Done in 49aba50de084dacdc65d1252e37fa83ec7b81574